time_interp/time_interp_external.F90

```diff
@@ -137,7 +137,8 @@ module time_interp_external_mod
   type(ext_fieldtype), save, private, pointer :: field(:) => NULL()
   type(filetype),      save, private, pointer :: opened_files(:) => NULL()
 !Balaji: really should use field%missing
-  real(DOUBLE_KIND), private, parameter :: time_interp_missing=-1e99
+  integer, private, parameter :: dk = DOUBLE_KIND
+  real(DOUBLE_KIND), private, parameter :: time_interp_missing=-1e99_dk
```

Comment:
if `DOUBLE_KIND` = 4, then won't this still cause an overflow?  Am I missing something here?

Reply:
You are correct.  However, `DOUBLE_KIND` should not equal 4 in any of our cases.